components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksInterceptor.kt

```diff
@@ -75,6 +75,7 @@ class AppLinksInterceptor(
 
         val doNotIntercept = when {
             uriScheme == null -> true
+            (!hasUserGesture && isSubframeRequest) -> true
```

Comment:
nit: Let's add a comment here to explain what this does: // We don't want to go to an external app for subframe requests not triggered by the user or similar.

Reply:
Agreed.  Thanks,